### ISSUE TITLE
Manually list sidebar links to pages

### ISSUE THREAD

**Issue by jbranchaud** [jbranchaud/splitting-atoms#56]
The sidebar has a set of links after the _Home_ and _Blog_ links that are automatically injected by Jekyll (those that are Jekyll pages). In order to have only specific links listed there and also make sure they are in a particular order, they should just be manually listed. I suggest the sidebar links the following pages in this order:
- About
- Introducing Atom
- Configuring Atom
- Developing Packages
- Developing Themes
- Best Practices
- Tips and Tricks
- Advanced Topics /\* this page doesn't exist yet, but should */
- Guide Listing /\* this page doesn't exist yet, but should */

Are there things that don't fall under these links?

**Comment by ZucchiniZe** [comment]
Once I get home from school I think I'll takle this.

**Comment by DWiechert** [comment]
How about a search feature? Can Jekyll easily provide that or would that require some extra work?

**Comment by jbranchaud** [comment]
@DWiechert This is out of the scope of Jekyll's built-in functionality. If you open another issue on this though, we can discussion what a search solution might look like.

**Comment by ZucchiniZe** [comment]
Would these be like categories?

**Comment by jbranchaud** [comment]
@ZucchiniZe No, they are pages that mostly already exist. The are basically what I expect to be the most common starting points for a person that drops into the home page looking for help with Atom.

**Comment by ZucchiniZe** [comment]
Ok
